localgov_elections_reporting.install

```diff
@@ -181,3 +181,29 @@ function localgov_elections_reporting_update_9003(&$sandbox) {
   $charts_config->set('advanced.requirements.cdn', TRUE);
   $charts_config->save();
 }
+
+/**
+ * Ensures map uses auto-boxing.
+ */
+function localgov_elections_reporting_update_9004(&$sandbox) {
+  $module_path = \Drupal::getContainer()
+    ->get('module_handler')
+    ->getModule('localgov_elections_reporting')
+    ->getPath();
+
+  $config_to_update = [
+    'views.view.electoral_map',
+  ];
+
+  $install_dir = new FileStorage($module_path . '/config/install/');
+  foreach ($config_to_update as $config) {
+    $data = $install_dir->read($config);
+    $config_obj = \Drupal::configFactory()->getEditable($config);
+    $config_obj->setData($data);
+    $config_obj->save(TRUE);
+  }
+
+  Drupal::logger('localgov_elections_reporting')->notice("Updated map to use auto-boxing");
+
+  drupal_flush_all_caches();
```

Comment:
I don't think we need to be this explicit.  `drush updatedb` clears the Drupal cache by default anyway.

Reply:
Have removed